src/KafkaFlow/KafkaFlow.csproj

```diff
@@ -10,7 +10,6 @@
         <PackageReference Include="Confluent.Kafka" Version="2.1.1" />
         <PackageReference Include="Microsoft.IO.RecyclableMemoryStream" Version="2.3.2" />
         <PackageReference Include="System.Threading.Channels" Version="6.0.0" />
-        <PackageReference Include="System.Memory" Version="4.5.5" />
```

Comment:
I think @filipeesch can shed some light here, if this is necessary or not.

Reply:
If the code is still compiling successfully and the samples and tests are working, we can safely remove it.